src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -46,6 +47,10 @@ public function validate($value, Constraint $constraint)
         $variables['value'] = $value;
         $variables['this'] = $this->context->getObject();
```

Comment:
to improve readability I would make this line an `else` branch of the `if` below